memorystore/redis/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/GoogleCloudPlatform/golang-samples/memorystore/redis
+
+go 1.13
+
+require github.com/gomodule/redigo v2.0.0+incompatible
```

Comment:
Looks like the redigo community has been spinning on what to do about this incompatible flag, possibly 2.0 is meant to be ignored.

Resolution in https://github.com/gomodule/redigo/issues/366 or https://github.com/gomodule/redigo/pull/440 are not clearly documented.

I wonder if it's since been fixed, and re-generating the go.mod and go.sum would help?

Reply:
Looks like it's fixed - I regenerated `go.mod` and `go.sum`